sentry_sdk/integrations/_wsgi_common.py

```diff
@@ -228,8 +218,7 @@ def _filter_headers(headers):
     }
 
 
-def _request_headers_to_span_attributes(headers):
-    # type: (dict[str, str]) -> dict[str, str]
+def _request_headers_to_span_attributes(headers: dict[str, str]) -> dict[str, str]:
```

Comment:
Sometimes it is `dict` sometimes `Dict`, we should make it consistent.

Reply:
different task